lsp-tailwindcss.el

```diff
@@ -200,6 +200,12 @@ This is a undocumented setting, see https://github.com/tailwindlabs/tailwindcss-
   :group 'lsp-tailwindcss
   :package-version '(lsp-tailwindcss . "0.3"))
 
+(defcustom lsp-tailwindcss-class-attributes ["class" "className" "ngClass"]
+  "The HTML attributes for which to provide class completions, hover previews, linting etc."
+  :type 'lsp-string-vector
+  :group 'lsp-tailwindcss
+  :package-version '(lsp-tailwindcss . "0.4"))
```

Comment:
Thanks! all looks great except the version should be "0.3" since 0.3 is not tagged yet.

Reply:
Thanks for the review. Fixed!